lib/arclight/custom_document.rb

```diff
@@ -31,7 +31,7 @@ class CustomDocument < SolrEad::Document
     def to_solr(solr_doc = {})
       super
       solr_doc['id'] = Arclight::NormalizedId.new(eadid.first).to_s
-      arclight_field_definitions.each do |field|
+      [*arclight_field_definitions, *arclight_field_definitions_colls].each do |field|
```

Comment:
Hmm, I feel like this might be the point where we're adding unnecessary cognitive load just to appease the cops.  I would be 👍 to excluding this file from the LineLength cop `.rubocop.yml` (with a comment explaining why).